ci-operator/config/openshift/etcd-quorum-guard/openshift-etcd-quorum-guard-master.yaml

```diff
@@ -0,0 +1,41 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: '4.0'
+  namespace: ocp
+base_images:
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+canonical_go_repository: github.com/openshift/etcd-quorum-guard
+images:
+- from: base
+  to: etcd-quorum-guard
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+tests:
+- as: unit
```

Comment:
we don't have a Makefile in this repo yet so remove the `unit` and `verify` test blocks

Reply:
In which repo -- `release` or `etcd-quorum-guard`?  I added one to `etcd-quorum-guard` in the last PR.

Reply:
https://github.com/openshift/etcd-quorum-guard/blob/master/Makefile

I see the `verify` target but not the `unit` target

Reply:
Sorry, nm, the target is `test` :+1: